scripts/build_demand_profiles.py

```diff
@@ -109,15 +110,29 @@ def get_load_paths_gegis(ssp_parentfolder, config):
     ssp = config.get("load_options")["ssp"]
 
     load_paths = []
-    for continent in region_load:
-        load_path = os.path.join(
-            ssp_parentfolder,
-            str(ssp),
-            str(prediction_year),
-            "era5_" + str(weather_year),
-            str(continent) + ".nc",
-        )
-        load_paths.append(load_path)
+    load_dir = os.path.join(
+        ssp_parentfolder,
+        str(ssp),
+        str(prediction_year),
+        "era5_" + str(weather_year),
+    )
+
+    for ext in [".nc", ".csv"]:
+        avail_regions = [
+            cnt
+            for cnt in region_load
+            if os.path.exists(os.path.join(load_dir, cnt + ext))
+        ]
+        if len(avail_regions) == len(region_load):
+            for continent in region_load:
+                load_path = os.path.join(load_dir, continent + ext)
+                load_paths.append(load_path)
+            break
+
+    # if len(load_paths) == 0:
```

Comment:
To delete at the end

Reply:
Fixed the message and revised the checks. 

Generally, I'd keep here the warning in case if non-existing demand data are requested: the current demand limitations seems to be not clear enough, as some misunderstandings are quite common in how to define `weather_year` and `prediction_year`. Although, a more detailed usability check could make sense. What do you think?